src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -202,6 +203,15 @@ public IObjectModelValidator ObjectValidator
         [NonAction]
         public virtual StatusCodeResult StatusCode(int statusCode)
             => new StatusCodeResult(statusCode);
+        
+        /// <summary>
+        /// Creates a <see cref="StatusCodeResult"/> object by specifying a <paramref name="statusCode"/>.
+        /// </summary>
+        /// <param name="statusCode">The status code to set on the response.</param>
+        /// <returns>The created <see cref="StatusCodeResult"/> object for the response.</returns>
+        [NonAction]
+        public virtual StatusCodeResult StatusCode(HttpStatusCode statusCode)
+            => new StatusCode((int)statusCode);
```

Comment:
Shouldn't have `new`?

> error CS0246: The type or namespace name 'StatusCode' could not be found (are you missing a using directive or an assembly reference?)

Reply:
Yeah this doesn't look quite right. I agree the `new` needs to be removed because it's just chaining a method call.

Reply:
Oops! Sorry, wrote it without a compiler based on something else I was working on. You're absolutely right.